src/Neo/Plugins/Plugin.cs

```diff
@@ -229,7 +247,33 @@ protected internal virtual void OnSystemLoaded(NeoSystem system)
         /// <returns><see langword="true"/> if the <paramref name="message"/> is handled by a plugin; otherwise, <see langword="false"/>.</returns>
         public static bool SendMessage(object message)
         {
-            return Plugins.Any(plugin => plugin.OnMessage(message));
+
+            return Plugins.Any(plugin =>
```

Comment:
I prefer using `if (Plugins.Count > 0)`, linq will be slower then using `Count`. `Any` iterates each item in the list.